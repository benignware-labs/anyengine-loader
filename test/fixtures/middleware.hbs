---
title: Bar
---
{{#> partials/layout.hbs}}
<h1>{{ title }}</h1>
{{/partials/layout.hbs}}
