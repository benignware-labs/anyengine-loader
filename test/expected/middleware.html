<html>
  <head>
    <title>Foobar | Bar</title>
  </head>
  <body>
    <h1>Bar</h1>
  </body>
</html>
